frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -116,6 +116,7 @@ export const DEFAULT_MDE = 30
 
 export interface ExperimentLogicProps {
     experimentId?: Experiment['id']
+    action?: string | null
```

Comment:
I'd suggest to type this explicitly to prevent invalid values:

```suggestion
    action?: 'create' | 'duplicate' | 'update'
```

Reply:
fixed in b9c8485fb3669f8b9eb10647f77576c3c3d54e59.